docs/command-reference.md

```diff
@@ -681,6 +681,7 @@ Flags:
   - :whale: `type=image,name=example.com/image,push=true`: Push to a registry (see [`buildctl build`](https://github.com/moby/buildkit/tree/v0.9.0#imageregistry) documentation)
 - :whale: `--progress=(auto|plain|tty)`: Set type of progress output (auto, plain, tty). Use plain to show container output
 - :whale: `--secret`: Secret file to expose to the build: id=mysecret,src=/local/secret
+- :whale: `--allow`: Allow extra privileged entitlement, e.g. network.host, security.insecure
```

Comment:
Maybe have a link to how to configure the entitlements
https://github.com/moby/buildkit/blob/master/docs/buildkitd.toml.md

Reply:
Thanks @AkihiroSuda 

The docs has been added :-)